### PR TITLE
Feature/music player

### PR DESCRIPTION
**Pull request by Pilkas-git**
Initial Music player feature. Works as it worked in windows forms. Also has a feature to take songs from web urls (will be needed as spotify provides song preview url). For spotify integration we still need to redesign the main menu window to support some kind of options or side pannel, because floading the user with buttons won't do any good.

### CODE REVIEW

**Review by tmaciulis22 — CHANGES_REQUESTED** [review]
Resolve conflicts

File: app-android-xamarin/Moodis/Moodis/Feature/CameraFeature/CameraActivity.cs [hunk 1]
```diff
@@ -59,7 +52,7 @@ private void StartCamera()
             cameraFragment = new CameraFragment();
             SupportFragmentManager.BeginTransaction()
                 .Replace(Resource.Id.content_frame, cameraFragment)
-                .Commit();
+                .CommitAllowingStateLoss();
```
Comment (tmaciulis22) [inline app-android-xamarin/Moodis/Moodis/Feature/CameraFeature/CameraActivity.cs]: Why you changed from simple commit?

File: app-android-xamarin/Moodis/Moodis/Resources/layout/activity_menu.xml [hunk 2]
```diff
@@ -5,6 +5,7 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:id="@+id/menuActivity"
```
Comment (tmaciulis22) [inline app-android-xamarin/Moodis/Moodis/Resources/layout/activity_menu.xml]: Put id attribute on the top, so xml file would be easier to read

File: app-android-xamarin/Moodis/Moodis/Resources/values/strings.xml [hunk 3]
```diff
@@ -35,9 +35,18 @@
 
   <string name="loading">loading</string>
 
+  <string name="song_already_playing">A song is already playing</string>
+  <string name="warning_in_request">Something wrong happened. Is your internet turned on?</string>
```
Comment (tmaciulis22) [inline app-android-xamarin/Moodis/Moodis/Resources/values/strings.xml]: unused string

**Review by tmaciulis22 — CHANGES_REQUESTED** [review]
🔍

File: app-android-xamarin/Moodis/Moodis/Feature/CameraFeature/CameraActivity.cs (hunk 1 above)
Comment (Pilkas-git) [reply app-android-xamarin/Moodis/Moodis/Feature/CameraFeature/CameraActivity.cs]: was required while testing on emulator.

**Review by tmaciulis22 — APPROVED** [review]

**Review by AArlauskas — APPROVED** [review]
Maladecas esi kad tvarka kode palaikai

**Comment by Pilkas-git** [comment]
how ?

**Pull request merged**